vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -5,50 +5,52 @@ sidebarDepth: 2
 
 ## Overview
 
-The Entando Command Line Interface (CLI) provides a set of commands that accelerate the developer experience by assisting the developer with common tasks such as quickly installing a new copy of Entando, generating an Entando project via JHipster, deploying an Entando Bundle, etc.
+The Entando Command Line Interface (CLI) provides a set of commands that accelerate the developer experience by assisting with common tasks such as installing a new copy of Entando, generating an Entando project via JHipster, deploying an Entando bundle, etc.
```

Comment:
Sorry, can we take out, 'by assisting'? there are 3 verbs before you get to the object and I think it works with 
"accelerate the developer experience with common tasks such as ..."

Reply:
i agree. i wasn't happy with this. i'm not sure that "accelerate the developer experience with common tasks" reads well? what about "... set of commands that assist the developer by accelerating common tasks such as..." or "... set of commands that accelerate common tasks such as..." assistance is kind of implied, i think...

Reply:
"the Entado Command Line Interface (CLI) provides a set of commands that helps the developer with common tasks such as ..."

Reply:
@jyunmitch i had changed it to "The Entando Command Line Interface (CLI) provides a set of commands that accelerate common tasks such as installing a new copy of Entando, generating an Entando project via JHipster, deploying an Entando bundle, etc."... i liked this variation the best. no good?

Reply:
It's good!